examples/boa/approach_env.go

```diff
@@ -57,7 +59,8 @@ func (ev *Approach) Desc() string {
 // Defaults sets default params
 func (ev *Approach) Defaults() {
 	ev.TimeCost = 0.05
-	ev.Acts = []string{"Forward", "Left", "Right", "Consume"}
+	ev.Acts = []string{"Forward", "Left", "Right", "Consume", "None"}
```

Comment:
I like the None action solution

Reply:
ps. it is not rendering "None" as nothing (zeros) on obelisk, but works here..

Reply:
Oh what is it rendering as? I can take a look at it.